### ISSUE TITLE
Unable to connect when running Octoprint 1.4.2 boot to browser.

### ISSUE THREAD

**Issue by Will-wastelander** [BillyBlaze/OctoPrint-TouchUI#439]
I know there was a previous issue reported about the same problem, and it looks like it has been closed. I just did an install of Octopi 0.17.0, and current TouchUI, and I am having the issue of "[::1] refused to connect." I have attempted to use the fix by changing localhost to be rPi's hostname, and this did not work either. I did a reinstall of 0.17.0, which comes with Octoprint 1.3.12, and TouchUI works w/o issues. Has there been any update on this bug ?

If I need to provide additional info to assist with troubleshooting or debugging, please let me know what to provide, and I will work on getting the info.

**Comment by Will-wastelander** [comment]
This was the previous bug report.

https://github.com/BillyBlaze/OctoPrint-TouchUI/issues/368

**Comment by Will-wastelander** [comment]
I have also updated python from 2.x to 3, but this issue existed before the python upgrade.

**Comment by Will-wastelander** [comment]
Oh, and I have disabled IPv6 w/ sysctl.conf using this page's info. https://cwesystems.com/?p=231

**Comment by Will-wastelander** [comment]
I found the resolution to this issue.

run the following commands after you have updated to 1.4.2 I did this, and it fixed my issue. This appears to be something that should be added to the installer script. @BillyBlaze 

SSH to Octopi.
`
~/oprint/bin/octoprint config set --bool server.allowFraming true ; sudo service octoprint restart ; sudo service touchui  restart
`

https://community.octoprint.org/t/my-touchui-display-is-no-longer-working-since-updating-to-octoprint-1-4-0/16819
